.github/workflows/create-rc-tag.yml

```diff
@@ -0,0 +1,121 @@
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18.x
+      - name: Get New Package Version
+        uses: ./.github/workflows/get-new-package-version
+        with:
+          CHANNEL: ${{ github.event.inputs.channel }}
+          TYPE: ${{ env.TYPE }}
+      - name: Bump Channel Version
+        uses: ./.github/workflows/bump-channel-version
+        with:
+          MAINTAIN_USERNAME: ${{ secrets.MAINTAIN_USERNAME }}
+          MAINTAIN_EMAIL: ${{ secrets.MAINTAIN_EMAIL }}
+          REPO_NAME: ${{ github.event.repository.name }}
+          CLEAN_PACKAGE_VERSION: ${{ env.PACKAGE_VERSION_CLEAN }}
+          CHANNEL: ${{ github.event.inputs.channel }}
+      - name: Install Dependencies
+        run: |
+          npm ci
+      - name: Build plugin
+        uses: ./.github/workflows/build-plugin
+        with:
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+          BUILD_SCRIPT_PATH: "./.github/scripts/build-zip.sh"
+          UPDATE_VERSIONS_SCRIPT_PATH: ""
+      - name: Upload zip file to GitHub actions artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.PLUGIN_ZIP_FILENAME }}
+          path: ${{ env.PLUGIN_ZIP_PATH }}
+          if-no-files-found: error
+      - name: Create Git tag
+        uses : ./.github/workflows/create-git-tag
+        with:
+          MAINTAIN_USERNAME: ${{ secrets.MAINTAIN_USERNAME }}
+          MAINTAIN_EMAIL: ${{ secrets.MAINTAIN_EMAIL }}
+          MESSAGE: v${{ env.PACKAGE_VERSION }}
+      - name: Get Previous Release
+        uses : ./.github/workflows/get-previous-release
+        with:
+          CHANNEL: ${{ github.event.inputs.channel }}
+      - name: Generate changelog
+        uses : ./.github/workflows/generate-changelog
+        with:
+          TOKEN: ${{ secrets.MAINTAIN_TOKEN }}
+          REPOSITORY: ${{ github.repository }}
+          HEAD_BRANCH_NAME: v${{ env.PACKAGE_VERSION }}
+          BASE_TAG_NAME: ${{ env.PREVIOUS_TAG_SHA }}
+      - name: Set empty changelog if no changes
+        run: |
+            # upload-artifact action fails if the file size is 0.
+            
+            # Check the file size
+            size=$(wc -c < "${{ env.CHANGELOG_FILE }}")
+            if [ $size -eq 0 ]; then
+              echo "No changelog" > "${{ env.CHANGELOG_FILE }}"
+            fi
+            
+      - name: Upload changelog to GitHub actions artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.CHANGELOG_FILE }}
+          path: ${{ env.CHANGELOG_FILE }}
+          if-no-files-found: error
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: v${{ env.PACKAGE_VERSION }}
+          files: |
+            elementor-*.zip
+            ${{ env.CHANGELOG_FILE }}
+          prerelease: true
+          body_path: ${{ env.CHANGELOG_FILE }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.MAINTAIN_TOKEN }}
```

Comment:
See: https://github.com/elementor/elementor-cloud-wp-agent/blob/master/.github/workflows/release.yaml#L24
and remove usage of `create-git-tag`

Reply:
Please pay attention to this message on the action repo : 
`This repository has been archived by the owner on Mar 4, 2021. It is now read-only.`

This action also don't support `files` Property.

The owner also recommends using the existing action that I'm already using : 
[softprops/action-gh-release](https://github.com/softprops/action-gh-release)

Reply:
We agreed to use the same method in the new gh-action `softprops/action-gh-release`